projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -366,4 +369,33 @@ public static function load_custom_css() {
 	public static function load_wpcom_random_redirect() {
 		require_once __DIR__ . '/features/random-redirect/random-redirect.php';
 	}
+
+	/**
+	 * Load the Starter Page Templates feature.
+	 */
+	public static function load_starter_page_templates_feature() {
+		// Disable the feature in ETK plugin
```

Comment:
Hmm what do you think of putting all this checks inside `Starter_Page_Templates::__construct()` instead, for consistency? This way, everything inside `load_etk_features()` are just `require_once ... ` statements nicely.

Reply:
We can put that at the last line of the `class-starter-page-templates.php`, after the class definition. I think I've seen this pattern before in our wpcom codebase, but can't tell at the top of my head 🤔

Reply:
Done in https://github.com/Automattic/jetpack/pull/38475/commits/b5acc02a8dce0b78cd86a78ca6ee154edcd71b9a